src/utils/converters.ts

```diff
@@ -0,0 +1,49 @@
+import { Converter, Coordinates, Size, Space } from '../types'
+
+import { norm } from './maths'
```

Comment:
Don't bring the WHOLE code of graph here. Just the minimal part that you need. This is probably the functions, and the basic classes. You just need an object able to store the details about an element, and make basic calculations about it (like orthoProjection)